app/src/main/java/com/eis/geoCalendar/network/SMS/SMSNetworkEventUser.java

```diff
@@ -0,0 +1,37 @@
+package com.eis.geoCalendar.network.SMS;
+
+import com.eis.geoCalendar.network.NetworkEventUser;
+import com.eis.smslibrary.SMSPeer;
+
+/**
+ * Simple class implementing the NetworkEventUser interface, that represents a user of an SMS Network
+ * @author Turcato
+ */
+public class SMSNetworkEventUser implements NetworkEventUser<SMSPeer> {
+    private SMSPeer myPeer;
+
+    /**
+     * Public constructor that creates a fully operative SMSNetworkEventUser object
+     *
+     * @param myPeer A valid SMSPeer
```

Comment:
Maybe it would be useful to put {@link SMSPeer}, as it belongs to a custom library.